compiler/damlc/tests/daml-test-files/TextMap.daml

```diff
@@ -6,7 +6,7 @@
 -- @INFO Use /=
 -- @INFO Use /=
 -- @INFO Use /=
-
+-- @INFO Avoid lambda
```

Comment:
Similar comment as for the eta reduce rule. I don’t think encouraging point-free style is helpful for beginner DAML programmers.

Reply:
Globally disabled

Reply:
I think point-free style is almost never good. 😸